pkg/proxy/iptables/proxier.go

```diff
@@ -569,6 +571,40 @@ func (proxier *Proxier) OnEndpointsDelete(endpoints *v1.Endpoints) {
 func (proxier *Proxier) OnEndpointsSynced() {
 	proxier.mu.Lock()
 	proxier.endpointsSynced = true
+	proxier.setInitialized(proxier.servicesSynced && proxier.endpointSliceSynced)
```

Comment:
What happens when the feature is gated off?  Will proxier.endpointSliceSynced ever be true?